stylesheets/about.css

```
@@ -0,0 +1,161 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Berkshire+Swash&family=Great+Vibes&display=swap');
+
+* {
+
+    margin: 0px;
+    padding: 0px;
+    font-family: 'Poppins', sans-serif;
+}
+
+body {
+    font-family: sans-serif;
+    text-align: center;
+    background-color: #dae1e7;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
+
+header {
+    background-color: lightblue;
+    padding: 1rem;
+}
+
+.navbar {
+    width: 100%;
+    color: #fff;
+    background-color: #5EBEC4;
+    height: 55px;
+}
+
+.navbar .container {
+    max-width: 1100px;
+    margin: 0px auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 70px;
+}
+
+.logo {
+    font-family: 'Berkshire Swash';
+    font-size: x-large;
+    color: #F92C85
+}
+
+span {
+    color: #fff;
+    font-family: 'Berkshire Swash', cursive;
```

Minor note - here you are setting the font-family on the parent (.logo) and here in the span.  One thing that you might consider is changing this to [inherit](https://developer.mozilla.org/en-US/docs/Web/CSS/inherit).  One of the nice things about using inherit here is that if you want to test with different fonts, you only have to change the font on .logo, and it will show up for both sections:
```suggestion
    font-family: inherit;
```

---

Thank you, will try inherit.